Kiss.Bff/Intern/ContactmomentDetails/Features/ContactmomentDetailsToevoegen.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Kiss.Bff.Beheer.Data;
+using Kiss.Bff.Intern.ContactmomentDetails.Data.Entities;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace Kiss.Bff.Intern.ContactmomentDetails.Features
+{
+    [ApiController]
+    public class ContactmomentDetailsToevoegen : ControllerBase
+    {
+        private readonly BeheerDbContext _db;
+
+        public ContactmomentDetailsToevoegen(BeheerDbContext db)
+        {
+            _db = db;
+        }
+
+        [HttpPut("/api/contactmomentdetails")]
+        public async Task<IActionResult> Post(ContactmomentDetailsModel model, CancellationToken cancellationToken)
+        {
+            var entity = await _db.ContactMomentDetails.Include(x => x.Bronnen).FirstOrDefaultAsync(x => x.Id == model.Id, cancellationToken: cancellationToken);
+
+            if (entity == null)
```

Comment:
je hebt alle velden uitgeschreven. de oude vorm (met uitzondering van de bronnen) werkte niet?

 ```
if (existingModel == null)
            {
                await _db.AddAsync(model, cancellationToken);
            }
            else
            {
                _db.Entry(existingModel).CurrentValues.SetValues(model);
            }
```

Reply:
Heb jij een voorkeur voor de oude vorm? Dat leek me ingewikkeld met navigation properties maar ik heb het niet getest.

Reply:
nou als het gewoon werkt met CurrentValues.SetValues dan is dat minder foutgevoelig. vroeg me vooral af of het bewust  was gewijzigd of dat de oude variant door verplaatsten van code uitzicht was geraakt.

Reply:
besproken. laten we zo